FirebaseSessions/Tests/TestApp/Podfile

```diff
@@ -0,0 +1,28 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+def shared_pods
+  pod 'FirebaseCore', :path => '../../../'
+  pod 'FirebaseInstallations', :path => '../../../'
+  pod 'FirebaseCoreInternal', :path => '../../../'
+  pod 'FirebaseCoreExtension', :path => '../../../'
+  pod 'FirebaseSessions', :path => '../../../'
+end
+
+target 'AppQualityDevApp_iOS' do
+  platform :ios, '15.0'
```

Comment:
Should we allow for older versions of iOS at all? May be a good reference for minimum compatible version for us?

Reply:
That's a good idea - set it out our deployment target in the cocoapod